tanner/sessions/session_analyzer.py

```diff
@@ -8,6 +8,8 @@
 from tanner.dorks_manager import DorksManager
 from tanner.config import TannerConfig
 
+COOKIE_INSERT_QUERY = "INSERT INTO cookies(sess_uuid, key, value) VALUES({uuid}, {k}, {v})"
```

Comment:
why it's here?

Reply:
oh this is just while testing. I'll remove the query or will use sqlalchemy for the execution of the query

Reply:
Ok :) please use something like TODO to highlight the parts which are still work in progress :)